ui_framework/doc_site/src/views/accessibility/screen_reader.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default () => (
+  <div>
+    <p>
+      This is the first paragraph. It is visible to all.
+    </p>
+    <p className='kuiScreenReaderOnly'>
```

Comment:
Thanks for the tip! That doesn't look too hard at all. It's nice that it can still be a functional component.

I don't mind adding this change to this PR.